test/pkg/environment/common/environment.go

```diff
@@ -158,9 +158,7 @@ func (env *Environment) DefaultNodeClass() *v1alpha1.KWOKNodeClass {
 func (env *Environment) DefaultNodePool(nodeClass *v1alpha1.KWOKNodeClass) *v1beta1.NodePool {
 	nodePool := test.NodePool()
 	nodePool.Spec.Template.Spec.NodeClassRef = &v1beta1.NodeClassReference{
-		Name:       "default",
-		Kind:       "KWOKNodeClass",
-		APIVersion: v1alpha1.SchemeGroupVersion.Version,
+		Name: nodeClass.Name,
```

Comment:
Don't we want kind and group/apiversion included?

Reply:
Done